src/compile.ts

```diff
@@ -120,3 +114,4 @@ const compileSchema = (filename: string) => {
 
 compileSchema('workflow-template-schema.yml');
 compileSchema('workflow-step-template-schema.yml');
+compileSchema('workflow-event-schema.yml');
```

Comment:
On a side note, the workflow schemas are currently only indirectly exposed via `dist/validation.js`, which requires one to call the exported `validate` function to retrieve the compiled schema. The event schema would not be accessible by the main entry point specified in the `package.json`.

It would probably be a good idea to have the schemas available without invoking `validate`, wdyt?

Reply:
I don't know how that would work. 
From the server perspective, we only use the yml files.

Reply:
> From the server perspective, we only use the yml files.

I suppose this works because of the submodule approach, or how do you access the compiled schemas? If we were to use this as a regular dep, there would only be one default export, which is `validation.js`.

I suppose we need an index file as the `main` entry point, which includes named exports for the schemas to make this perfectly reusable for all use cases.

Reply:
Good point! Now I see. 
I'll address it